### PR TITLE
Core/Misc: rename namespaces and macros to acore

### PR DESCRIPTION
**Pull request by Viste**
#### CHANGES PROPOSED:
-  give proper name for namespaces and macros there's no reason to be trinity when you azeroth 
##### Target branch(es):
- [x] Master

### CODE REVIEW

File: src/server/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -7741,10 +7741,10 @@ bool ObjectMgr::LoadTrinityStrings()
     uint32 oldMSTime = getMSTime();
 
     _trinityStringStore.clear(); // for reload case
-    QueryResult result = WorldDatabase.PQuery("SELECT entry, content_default, content_loc1, content_loc2, content_loc3, content_loc4, content_loc5, content_loc6, content_loc7, content_loc8 FROM trinity_string");
+    QueryResult result = WorldDatabase.PQuery("SELECT entry, content_default, content_loc1, content_loc2, content_loc3, content_loc4, content_loc5, content_loc6, content_loc7, content_loc8 FROM ACORE_string");
     if (!result)
     {
-        sLog->outString(">> Loaded 0 trinity strings. DB table `trinity_strings` is empty.");
+        sLog->outString(">> Loaded 0 trinity strings. DB table `ACORE_strings` is empty.");
```
Comment (FrancescoBorzi) [inline src/server/game/Globals/ObjectMgr.cpp]: tables name should be all lowercase

Comment (Viste) [reply src/server/game/Globals/ObjectMgr.cpp]: just a VSCode mass rename )

**Comment by Viste** [comment]
For pass build need https://github.com/ElunaLuaEngine/Eluna/pull/299

**Comment by Viste** [comment]
https://github.com/azerothcore/mod-transmog/pull/28

**Comment by Viste** [comment]
https://github.com/azerothcore/mod-eluna-lua-engine/pull/20

**Comment by Stoabrogga** [comment]
Instead of just removing the module script from Travis I added comments in order to provide information about what's going on here.

**Comment by Viste** [comment]
> Instead of just removing the module script from Travis I added comments in order to provide information about what's going on here.

Thank you, that was very helpful!

**Comment by Viste** [comment]
> Instead of just removing the module script from Travis I added comments in order to provide information about what's going on here.

Thank you, that was very helpful.

**Comment by FrancescoBorzi** [comment]
@Viste @Winfidonarleyan has anyone tested this? can it be merged?

**Comment by FrancescoBorzi** [comment]
Tested:

- Build, run
- Log in game, try to kill some NPCs and level up 
- Tried chat
- Log out, log in

Everything looks normal. Ready to be merged.

**Pull request merged**